client/package.json

```diff
@@ -90,7 +90,7 @@
     "jest-environment-jsdom": "^29.3.1",
     "jest-enzyme": "^7.1.2",
     "moxios": "^0.4.0",
-    "prettier": "^1.19.1",
+    "prettier": "^2.8.4",
```

Comment:
There's a fair amount of the existing formatting that's treated as errors due to the upgrade to `prettier`, that I think this PR could address as well. The changes can be seen when using the existing script command `yarn prettier` but that will write the changes. You can also check them with `npx eslint .`:

```
  ...
  102:29  error  Insert `,`                                          prettier/prettier
  109:25  error  Insert `,`                                          prettier/prettier
  155:23  error  Insert `,`                                          prettier/prettier

/path/to/aria-at-app/server/util/getUsersFromFile.js
   4:32  error  Replace `file` with `(file)`    prettier/prettier
   8:40  error  Replace `line` with `(line)`    prettier/prettier
   9:40  error  Replace `(line` with `((line)`  prettier/prettier
  10:35  error  Replace `line` with `(line)`    prettier/prettier

/path/to/aria-at-app/server/util/sortArrayLikeArray.js
  16:20  error  Replace `array2Item` with `(array2Item)`  prettier/prettier
  17:24  error  Replace `array1Item` with `(array1Item)`  prettier/prettier

/path/to/aria-at-app/server/util/sortArrayLikeArray.test.js
   9:38  error  Insert `,`                                                           prettier/prettier
  15:32  error  Replace `item·=>·item.id·||·item` with `(item)·=>·item.id·||·item,`  prettier/prettier
  22:41  error  Insert `,`                                                           prettier/prettier

✖ 1743 problems (1743 errors, 0 warnings)
  1743 errors and 0 warnings potentially fixable with the `--fix` option.
```

Updating `.prettierrc` with the following props gets us closer to what existed before:
```
"arrowParens": "avoid",
"trailingComma": "none"
```

But the formatting warnings that remain are opinionated whitespace handling coming from prettier which I have no real stance on. They'll also conflict with the default `eslint` rules.

Reply:
@howard-e I added the prettier changes. There might be potential workarounds if there are conventions we want to keep.

Reply:
@alflennik I'm ok with doing that. In this PR @howard-e?

Reply:
Oh, got it!